src/backends/onnxruntime.c

```diff
@@ -206,8 +237,8 @@ RAI_Model *RAI_ModelCreateORT(RAI_Backend backend, RAI_Device device,
   // loading a model from a buffer, we'll keep this as is
   char tmpfile[256];
   FILE *fp;
-  snprintf(tmpfile,256,"temp-%d.onnx", (int) getpid());
-  fp = fopen(tmpfile,"w");
+  snprintf(tmpfile, 256, "temp-%d.onnx", (int) getpid());
```

Comment:
why not use mkstemp?

Reply:
Since the command is executed on the main thread, I'm trying to avoid having too many temp files around. BTW, as soon as the new release for ONNXRuntime is out, we can remove this section of the code. Hopefully it'll be before GA